### ISSUE TITLE
Click on empty taskbar space - Classic taskbar on Windows 11 not working on secondary taskbars

### ISSUE THREAD

**Issue by fehmi1907** [m1lhaus/windhawk-mods#10]
Running Windows 11 23H2 with use enhanced classic taskbar option in StartAllBack 3.7.5. Click on empty taskbar space works great on the primary taskbar once I enable use the old taskbar on Windows 11 option in the mod settings; however, nothing happens on the secondary taskbars when I double or middle click.

**Comment by TheHungryOne** [comment]
Assuming that by saying "secondary taskbars" you meant the taskbar on non-primary display when using multiple displays \ monitors, I've tested it too,

For me double-click opens snipping tool app and middle click is set to show my desktop.
The "start application" feature seems to be working properly on all displays but the "show desktop" feature only works on my primary display,

**Comment by fehmi1907** [comment]
To better clarify, I am referring to taskbars on non-primary displays. The actions I use are _Task Manager_ and _Show desktop_.

@TheHungryOne After seeing your comment, I tried assigning different actions to see if they work on secondary taskbars.

Works:
- Start application
- Ctrl+Alt+Tab
- Taskbar auto-hide
- Win+Tab
- Open Start menu

Does not work:
- Show desktop
- Task Manager
- Mute system volume

As a workaround for Task Manager, I assigned it as a custom executable in start an application.
